File: Array/Tasks/task.go
package main

import "fmt"

// Create a Go program that defines a slice of integers, then creates a function that takes a pointer to the slice and modifies it to include only the odd numbers

func evenize(slice *[]int) []int {
	var filtered_slice []int = []int {}

	for _,item := range *slice {
		if item % 2 == 0 {
			filtered_slice = append(filtered_slice, item) 
		}
	}
	return filtered_slice
}

func main() {
	var numbers []int = []int{1, 2, 3, 4, 5, 6}

	var even_numbers []int = evenize(&numbers)

	fmt.Print("Slice containing only even numbers:", even_numbers)
}

File: Array/Tasks/task.py
def evenize_list(list: list[int]):
    even_numbers = []
    for number in list:
        if number % 2 == 0:
            even_numbers.append(number)
    return even_numbers

def evenize_list_2(list: list[int]):
    return [item for item in list if item % 2 == 0]

numbers = [1,5,2,9,7,6,3,22,55,44,68]

# even_numbers = evenize_list(numbers)
even_numbers = evenize_list_2(numbers)

print(f"The even numbers in the provided list are: {even_numbers}")